intellij/testing/src/main/kotlin/motif/intellij/testing/IntelliJRule.kt

```diff
@@ -29,21 +27,6 @@ class IntelliJRule : TestRule {
 
       override fun evaluate() {
         TestApplicationManager.getInstance()
-        var e: Throwable? = null
-        val latch = CountDownLatch(1)
-        ApplicationManager.getApplication().invokeLater {
```

Comment:
This `invokeLater` hangs forever because `TestApplicationManager ` is preferred over `ApplicationManager`.

Reply:
What about the evaluation call of the base statement? Don't we skip the actual test without invoking it?

Reply:
I must admit I don't know enough about the original intention of the rule - but I tried breaking one of the test cases by messing with the annotation, and even with this rule change, the test fails.

Reply:
I see what you are saying now. I was certain that the test that ran was part of the `TestHarness`, but let me put back the `base.evaluate()` call and double-check - it does seem like the test is doing nothing right now.

Reply:
I think I fixed it in a better way in [a7ee27d](https://github.com/uber/motif/pull/256/commits/a7ee27d8eb2fa25729d2cfc91516bbdfb783d09f) - there are some odd interaction by having the setUp method annotated with `@Before` (and tearDown/`@After`) based on my observation and the code comment in the [ancestor class UsefulTestCase](https://github.com/JetBrains/intellij-community/blob/master/platform/testFramework/src/com/intellij/testFramework/UsefulTestCase.java#L91) - they seem to be called twice and making the test project unhappy.

Not sure that interaction was related to the intellij rule, but since the rule doesn't seem needed, I've also removed it in the same commit.

I have verified by running the test without the rule and commenting out the "addLibrary" calls in the setup and run TestHarness and noticing that only a handful of the test passed in the suite and the rest failed - thus giving me confidence that it's not skipping all the tests.